client/src/pages/learn/coding-interview-prep/take-home-projects/index.md

```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
 ---
 ## Introduction to the Coding Interview Take Home Projects
 
-When applying for a job as a developer, there's a good chance the hiring team will give you something called a "take-home project". As the name suggests, it's something that you're supposed to do on your own, without the interviewer looking over your shoulder. Unlike other parts of the coding interview, which often focus more on theoretical knowledge and less on actual app development - a take-home project is usually the best place to show your skills in practice.
+Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project which they can complete in their own time. These type of interviews usually require a lot of work but it is the most preferred interviewing process.
 
-In this section you'll be able to work on examples of such projects!
\ No newline at end of file
+The interview after project submission could be a discussion about applicant’s design decisions, scalability, and what they might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects to talk about.
```

Comment:
It should be `the applicant's design decisions,`  Again, you can not use the word `they` to refer to a single applicant, so `they` should not be used here.  Also, you never end a sentence with a preposition (i.e `about`).  Lastely, "out of the box" is an expression which may confused readers not familiar with such expressions.  Consider rephrasing it in "plain" English.  I don't have a particular suggestion off the top of my head, but I am confident you will think of something.  Note that my suggestion below does not add an alternative for you, but does correct grammar and spelling mistakes.
```suggestion
The interview after the project submission could be a discussion about the applicant’s design decisions, scalability, and what the applicant might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects.
```

Reply:
I'm sorry for my bad grammar! Thank You for the suggestions. @RandellDawson

Reply:
Looks good now.